src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -16,7 +15,7 @@ class ProjectCreationModel {
 //    var resource by resourceProperty
    // var resourceProperty = getProperty(ProjectCreationModel::resourceSelected)
 
-    private var project: ObservableList<Project> by property(
+    private var project: ObservableList<Collection> by property(
             FXCollections.observableList(ProjectList().projectList
             )
```

Comment:
unless the arguments are on their own line, put the close parentheses on the same line

Reply:
@sarabiaj The change here was to fix compile. The updated projectcreation models/views/etc. are on ns_new_wizard_usecase

Reply:
alright, cool